pycon/sponsorship/tests/test_forms.py

```diff
@@ -6,6 +9,10 @@
 
 from .factories import SponsorLevelFactory
 
+# Tiny image file for testing
+TEST_IMAGE_FILENAME = os.path.join(os.path.dirname(__file__), 'colormap.gif')
```

Comment:
I don't know that it matters, but you could generate this sort of file in a utility function. I personally prefer that to creating a particular file.